package.json

```diff
@@ -21,25 +21,30 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "@codegenie/serverless-express": "^4.14.0",
     "@nestjs/common": "^7.5.1",
     "@nestjs/config": "^0.5.0",
     "@nestjs/core": "^7.5.1",
     "@nestjs/jwt": "^7.2.0",
     "@nestjs/passport": "^7.1.0",
     "@nestjs/platform-express": "^7.5.1",
+    "aws-lambda": "^1.0.7",
     "helmet": "^4.2.0",
     "passport": "^0.4.1",
     "passport-http": "^0.3.0",
     "passport-jwt": "^4.0.0",
     "passport-local": "^1.0.0",
+    "pg": "^8.11.5",
     "reflect-metadata": "^0.1.13",
     "rxjs": "^6.5.4",
+    "serverless-dotenv-plugin": "^6.0.0",
```

Comment:
Seems like we don't need this plugin because env variables are now accessible by serverless itself using `usedotev: true` option. Please, take a look https://www.serverless.com/framework/docs/environment-variables